README.md

```diff
@@ -109,3 +113,5 @@ Markdown uses email style notation for blockquotes and I've been told:
 > Asterisks for *emphasis*. Double it up  for **strong**.
 
 `<?php code(); // goes in backticks ?>`
+
+This is a living breathing document, please treat it with care.
```

Comment:
if it's living and breathing, is it really a document? Maybe we should call it an organism?